From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: twitharshil <twitharshil@gmail.com>
Date: Tue, 11 Aug 2020 02:51:54 -0400
Subject: BoringSSl Tls Renegotiation patch


diff --git a/src/tls_wrap.cc b/src/tls_wrap.cc
index 42b9469e38189f04745732afdeadd59e3ce6ad4c..ab0fcd4751434ecd04b63719371ebe0d6b1bc0d8 100644
--- a/src/tls_wrap.cc
+++ b/src/tls_wrap.cc
@@ -145,6 +145,7 @@ void TLSWrap::InitSSL() {
   if (is_server()) {
     SSL_set_accept_state(ssl_.get());
   } else if (is_client()) {
+    SSL_set_renegotiate_mode(ssl_.get(), ssl_renegotiate_freely);
     // Enough space for server response (hello, cert)
     crypto::NodeBIO::FromBIO(enc_in_)->set_initial(kInitialClientBufferLength);
     SSL_set_connect_state(ssl_.get());
